content/projects/katas/level-1/_index.md

```diff
@@ -57,6 +57,15 @@ In other words, there should be no directories. Only modules.
 
 If you are writing JS code then your files would follow a similar pattern. You would just have `.js` instead of `.py` files.
 
+If you are using Java, your file names should be capitalized, like so:
+
+```
+├── Task1.java
+├── Task2.java
+└── Task3.java
+...
```

Comment:
For Task7 should we say that it could be a lists/array and specifying that they are of integer type